src/main/resources/digital/slovensko/autogram/ui/gui/settings-dialog.fxml

```diff
@@ -150,19 +150,17 @@
                                     <VBox styleClass="left">
                                         <TextFlow>
                                             <Text styleClass="autogram-heading-s">
-                                                Použitie nového štandardu
+                                                Automatické odpojenie pri nečinnosti
                                             </Text>
                                         </TextFlow>
                                         <TextFlow>
                                             <Text styleClass="autogram-description">
-                                                Podpisovanie štandardom ETSI EN 319 132 a&#160;ETSI EN 319 122.
-                                                Pozor, systémy verejnej správy tento štandard väčšinou nepodporujú.
+                                                Po koľkých minútach nečinnosti bude s kartou zrušené spojenie.
                                             </Text>
                                         </TextFlow>
                                     </VBox>
-                                    <VBox styleClass="autogram-checkbox-container">
-                                        <HBox fx:id="en319132Radios"
-                                            styleClass="autogram-smaller-radio-buttons" />
+                                    <VBox styleClass="autogram-textfield-container,autogram-textfield-container--width-3">
+                                        <TextField fx:id="tokenSessionTimeoutTextField" styleClass="autogram-input" />
```

Comment:
snazim sa drzat tych gov uk nazvov stylov aby som to nemusel hladat. Cize tuto by som to dal priamo ako class `autogram-input--width-3` na ten `TextField` element a nie ten nad nim.